setup.md

```diff
@@ -0,0 +1,156 @@
+
+First, if you don't already have the AWS CLI installed, follow the instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+
+### Marine mammal audio samples
+
+We have downloaded the audio files from the [Watkins Marine Mammal Sound Database](https://cis.whoi.edu/science/B/whalesounds/fullCuts.cfm) using the [/orca_detector/getData.py](./orca_detector/getData.py) script, based on a sample provided to us by Watkins.
+
+Expand `data.tar.gz` inside the main directory of the repo; it will create a `/data` folder.  (There are `.gitignore` rules in place to prevent it from accidentally being submitted to GitHub.)
+
+```
+cd ~/OrcaDetector
+aws s3 cp s3://w251-orca-detector-data/data.tar.gz .
+tar -xvf data.tar.gz
+```
+
+Alternately you can download [data.zip](https://drive.google.com/file/d/10mGIptby8SEf4yk0m57mVtiXCgRQrgTc/view?usp=sharing) from Google Drive, but due to the size of the archive, it's probably much faster to pull it from the S3 bucket.
+
+### Pretrained weights (for VGGish Keras model)
+
+Download and extract `vggish_weights.tar.gz ` inside the main directory of the repo; it will create a `/vggish_weights` folder containing VGGish model weights that were pretrained on the [AudioSet](https://research.google.com/audioset/index.html) dataset, which contains audio from 2 million human-labeled 10-second YouTube clips.
+
+```
+cd ~/OrcaDetector
+aws s3 cp s3://w251-orca-detector-data/vggish_weights.tar.gz .
+tar -xvf vggish_weights.tar.gz
+```
+
+## 4. Create the `orca_dev` Docker image
+
+### GPU OPTION: Build our `orca_dev` base Docker image
+
+In the project repo, `cd` into the `orca_detector` directory:
+
+```
+cd ~/OrcaDetector/orca_detector
+```
+
+Build the Docker image (this will take a while):
+
+```
+sudo docker build -t orca_dev -f Dockerfile.dev .
+```
+
+## 5. Launch an `orca_dev` Docker container
+
+Run the `orca_dev` Docker container with the following args.  
+
+> NOTE: update the host volume mappings (i.e. `~/OrcaDetector`) as appropriate for your machine in the following script:
+
+```
+sudo docker run \
+    --rm \
+    --runtime=nvidia \
+    --name orca_dev \
+    -ti \
+    -e JUPYTER_ENABLE_LAB=yes \
+    -v ~/OrcaDetector/orca_detector:/src \
+    -v ~/OrcaDetector/data:/data \
+    -v ~/OrcaDetector/vggish_weights:/weights \
+    -v ~/OrcaDetector/results:/results \
+    -p 8888:8888 \
+    -p 4040:4040 \
+    orca_dev
+```
+
+You will see it listed as `orca_dev ` when you run `docker ps -a`.  
+
+### Verify Keras can see the GPU
+
+Once inside the container, try running:
+
+```
+nvidia-smi
+```
+
+### Verify that the `vggish` Keras model builds
+
+Once inside the container, the following script should run to make sure we can instantiate the Keras model and load it's pretrained weights:
+
+```
+python3 vggish_model.py
+```
+
+If it was successful, you should see a Keras model summary.
+
+
+### (REMOVE?) Verify that the `vggish` TF smoke test runs
```

Comment:
Well, I said above that we should remove the smoke test, but I'm liking the idea of an orca smoke test now..

Reply:
I'll file an issue to create our own and delete this one at that time.